app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,15 @@
+class RelationshipsController < ApplicationController
```

Comment:
Fix indentation for this file.
2 spaces for .rb files
2 spaces for .scss files
4 spaces for .html.erb files